pom.xml

```diff
@@ -188,21 +177,27 @@
     <!-- ======================================================================= -->
     <dependencies>
         <dependency>
-            <groupId>org.owasp.antisamy</groupId>
-            <artifactId>antisamy</artifactId>
-            <version>1.6.4</version>
+            <groupId>com.googlecode.owasp-java-html-sanitizer</groupId>
+            <artifactId>owasp-java-html-sanitizer</artifactId>
+            <version>20211018.2</version>
+            <scope>provided</scope>
```

Comment:
It is embedded via https://github.com/apache/sling-org-apache-sling-xss/blob/7d5b533f190ebf02a8c837546f2d230098733ecc/bnd.bnd#L46

Reply:
Can we switch to using either https://bnd.bndtools.org/instructions/includeresource.html (to embed the full jar) or https://bnd.bndtools.org/instructions/conditionalpackage.html for individual packages?

Reply:
The reason for embedding the bundle is that we're using reflection to remove some features which conflict with the way we want to use it, see for instance https://github.com/apache/sling-org-apache-sling-xss/blob/7d5b533f190ebf02a8c837546f2d230098733ecc/src/main/java/org/apache/sling/xss/impl/CustomPolicy.java#L243-L253 . 

The HTML Sanitizer insists on setting some sane defaults, which is good in most scenarios. On the other hand, we expect to configure everything from scratch using an AntiSamy config so we reconcile them by simply nuking their defaults.

If we allow the sanitizer library to be deployed as a bundle we run the risk of them changing the internals and breaking the way we access then.

Reply:
Also, yes - it would be nice to solve SLING-10398, but let's get this right first and we can do a quick follow-up later. This is large enough as it is :-)